lib/roo_on_rails/checks/multi.rb

```diff
@@ -0,0 +1,17 @@
+module RooOnRails
+  module Checks
+    class Multi
+      def initialize(variants:, of:)
+        @_variants = variants
+        @_of = of
```

Comment:
I don't really see the point of `@_` here, especially as this class isn't designed to be mixed, inherited, etc. What's wrong with just `@`? IMHO this convention makes the code needlessly harder to read.